### PR TITLE
Remove unused schema parameter and cleanup.

### PR DESCRIPTION
**Pull request by davinchia**
## What
Remove unused schema parameter for better readability.

## Pre-merge Checklist
- [ ] *Run integration tests*

### CODE REVIEW

File: airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java [hunk 1]
```diff
@@ -437,13 +428,11 @@ private static void assertColumnsWithSameNameAreSame(String schemaName, String t
   }
 
   @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
-  private List<TableInfoInternal> discoverInternal(final JdbcDatabase database,
-                                                   final Optional<String> databaseOptional,
-                                                   final Optional<String> schemaOptional)
+  private List<TableInfoInternal> discoverInternal(final JdbcDatabase database, final Optional<String> databaseOptional)
```
Comment (ChristopheDuong) [inline airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java:431]: I'm wondering if this `databaseOptional` is not unused as the schema parameter too? (depending on the source database, we sometimes use it as a schema replacement)

**Review by ChristopheDuong — APPROVED** [review]

File: airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java (hunk 1 above)
Comment (davinchia) [reply airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java:431]: I think you may be right. Will take a second look after!

**Pull request merged**